ci/scripts/manage-versions.mjs

```
@@ -0,0 +1,111 @@
+      process.exit(1);
+    }
+
+    updateCoreDependency(coreVersion);
+    result = {
+      transport_version: incrementTransportVersion(),
+      core_version: coreVersion,
+    };
+    break;
+
+  case "transport":
+    const transportVersion = inputVersion;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)